public/contribute/index_zh.ejs

```diff
@@ -1,3 +1,19 @@
+<%
```

Comment:
@bspencer , I agree with your comment. I have tried to put these functions in a utils.ejs and include them instead of putting them at the top of multiple files. But the error occurs that the function "languageDetect" not defined. I am a little confused about this. If I add some html elements into utils.ejs, they will show up in the file which include utils.ejs. However, the function still don't work if even though the utils.ejs has been included.